project/AppDependencies.scala

```diff
@@ -12,15 +12,16 @@ object AppDependencies {
     "io.circe"    %% "circe-core"                       % circeVersion,
     "io.circe"    %% "circe-generic"                    % circeVersion,
     "io.circe"    %% "circe-parser"                     % circeVersion,
-    "io.circe"    %% "circe-yaml"                       % circeVersion,
-    "io.circe"    %% "circe-generic-extras"             % circeVersion
+    "io.circe"    %% "circe-yaml"                       % "1.15.0"
```

Comment:
the versions are only coincidentally sometimes overlapping

Reply:
is it worth inlining the rest to make that explicit?  or having a comment here for future reference?

Reply:
yup

Reply:
I'll inline and comment in case they overlap again and someone thinks to refactor it

Reply:
some might be aligned, I'm not sure